src/display.js

```diff
@@ -1,14 +1,17 @@
-import {Board}  from './index.js'; // Module is evaluated!
 
+import { RefreshData } from './scoresAPI.js';
 const scoresBoard = document.getElementById('scores');
-const displayScores = () => {
+export const displayScores = async () => {
   scoresBoard.innerHTML = '';
-  // Boards.timeDisplay()
```

Comment:
- [ ] To make your project professional, kindly remove unused commented codes from your file.